src/App.jsx

```diff
@@ -17,8 +17,21 @@ function App() {
   const [correctAnswers, setCorrectAnswers] = useState(0);
   const [start, setStart] = useState(true);
   const [reset, setReset] = useState(true);
+  const [showResult, setShowResult] = useState(false);
+  const [token, setToken] = useState('');
+  // const [resCode, setResCode] = useState('');
 
-  const notify = () => toast.error('Error fetching data');
+  const { code, message } = RESPONSE_CODES;
+
+  let resCode;
+  let resMessage;
+
+  // const { code, message } = RESPONSE_CODES.find(
```

Comment:
remove commented code

if it's something valuable you're afraid to lose, then push it into a separate branch or stash it locally